test/fast/git-process-test.ts

```diff
@@ -359,5 +359,19 @@ remote: http://github.com/settings/emails`
       const error = GitProcess.parseError(stderr)
       expect(error).to.equal(GitError.OutsideRepository)
     })
+
+    it('can parse lock file exists error', () => {
+      const stderr = `Unable to create '%s.lock': %s.\n
```

Comment:
I think the use of backticks here means the `\n`s are unnecessary. See some of the longer `stderr` examples from above. 

Also, can we stub `%s` with something more relevant here from a real-world example of the error message?